tree/ntuple/v7/src/RFieldBase.cxx

```diff
@@ -594,7 +594,10 @@ ROOT::Experimental::RFieldBase::Create(const std::string &fieldName, const std::
 
       if (!result) {
          auto cl = TClass::GetClass(canonicalType.c_str());
-         if (cl != nullptr) {
+         // NOTE: if the class is not at least "Interpreted" we don't have enough information to
+         // properly construct the RClassField (e.g. we are missing the list of bases), so in that
```

Comment:
This is not precise.  The list of base class is available in the `TStreamerInfo` and thus available for an Emulated class.

Is this PR a temporary fix/workaround or am I missing something?

Reply:
How do you access that information? When we load a class from file without dictionary we observe that `TClass::GetListOfBases()` returns `nullptr`. Do we need to take some explicit steps to have that list filled? Or do we need to call a different method?

Reply:
You have to look at the TStreamerInfo, eg.
```
auto info = cl->GetStreamerInfo(); // This return the current StreamerInfo describing the in memory layout
for(auto elem : TRangeStaticCast<TStreamerElement>( *info ) {
   if (elem->IsBase()) {
       ...
   }
}
```

Reply:
@pcanal  I guess you meant `TRangeStaticCast<TStreamerElement>(info->GetElements())`; 
but this gives me `TStreamerElement`s and `TStreamerBase`s: how do I get to the `TBaseClass` from there?
Or can I still access stuff like `GetDelta()` from a `TStreamerBase`?

Reply:
>  I guess you meant TRangeStaticCast<TStreamerElement>(info->GetElements());

Indeed, sorry :)

> how do I get to the TBaseClass from there?

You can't, it is a different interface.

> Or can I still access stuff like GetDelta() from a TStreamerBase?

Yes, through `GetOffset`.  Note that the list of `TStreamerElement` will only contains the direct base classes.

Reply:
@pcanal for now we decided to go through field emulation when the TClass is emulated, we might change this in the future if we see the need for it

Reply:
You might still need to update the PR summary.

I still find it confusing to use the same adjective for to similar but distinct concept.  I.e. 'emulated [T]Class' vs 'emulated Field'.

Reply:
We can rediscuss the terminology, but I don't believe this PR is the best place to do so.
I personally don't have a problem with it, considering it's an analogous concept done in a different context. As long as both concepts are well documented, I don't think there is much room for confusion.

Reply:
Fair enough.